Bayesian_Cognitive_Modeling/ParameterEstimation/DataAnalysis/Kappa.stan

```diff
@@ -0,0 +1,55 @@
+// #### Notes to Stan model #######################################################
+// ## 1) This is the first time we use simplex data type. Simplex is similar to 
```

Comment:
Nope.  Just cut all this.  People use these models for reference, they don't read this stuff in order!  If they need to know what a simplex is, they should go read some doc.  The other stuff is all historical and just documenting the Stan language for which we have reference material.  If you were writing a doc and explaining to people who know  BUGS you might include *some* of this.